models/fhir/medication-dispense.ts

```diff
@@ -7,25 +7,24 @@ import * as medication from "./medication"
 
 export interface MedicationDispense extends common.Resource {
   resourceType: "MedicationDispense"
+  contained: Array<medicationRequest.MedicationRequest>
   identifier: Array<common.Identifier>
   extension?: Array<extension.CodingExtension | extension.ExtensionExtension<extension.IntegerExtension>>
   medicationCodeableConcept?: common.CodeableConcept
   medicationReference?: common.Reference<medication.Medication>
   subject: common.IdentifierReference<patient.Patient>
   quantity: common.SimpleQuantity
-  authorizingPrescription: Array<AuthorizingPrescription>
+  authorizingPrescription: AuthorizingPrescription
   whenHandedOver: string
   dosageInstruction: Array<medicationRequest.Dosage>
   performer: Array<DispensePerformer>
   type: common.CodeableConcept
 }
 
 export interface AuthorizingPrescription extends common.IdentifierReference<medicationRequest.MedicationRequest> {
```

Comment:
can we delete this interface now? just say that `authorizingPrescription: common.IdentifierReference<medicationRequest.MedicationRequest>` in the interface above?